site2/docs/admin-api-tenants.md

```diff
@@ -30,7 +30,8 @@ my-tenant-2
 
 ### Create
 
-#### pulsar-admin
+<!--DOCUSAURUS_CODE_TABS-->
```

Comment:
For this file, only `pulsar-admin` is listed, so it's not necessary to adopt switch tab.
You can try to refine the language from technical writing part to make it more clear.